transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -87,24 +87,22 @@ abstract class AbstractChannelHandlerContext implements ChannelHandlerContext, R
     private final String name;
     private final int executionMask;
 
-    // Cache the concrete value for the executor() method. This method is in the hot-path,
-    // and it's a profitable optimisation to avoid as many dependent-loads as possible.
-    // It does not need to be volatile, because it's always the same value for a given context,
-    // within the lifetime of its registration with an event loop, and deregistering will clear it.
-    EventExecutor contextExecutor;
```

Comment:
We don't need this anymore as we can just store it directly in a final field in 5.0 (every context use the same executor which never changes)